rush.json

```diff
@@ -23,6 +23,11 @@
       "packageName": "@microsoft/applicationinsights-react-js",
       "projectFolder": "applicationinsights-react-js",
       "shouldPublish": true
+    },
+    {
+      "packageName": "@microsoft/applicationinsights-react-sample",
```

Comment:
The tests are failing because of the `rush check` which validates that all components are using the same version of the imports.
You can also run locally to validate -- I get caught be this from time to time when I'm update component governance issues.